db/schema/item.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const user = require('./user');
+const transaction = require('./transaction')
+
+const itemSchema = new mongoose.Schema({
+    uuid: String,
+    transaction: transaction,
+    currency: {
+        type: String,
+        default: 'USD'
+    },
+    itemName: String,
+    people: [user],
+    cost: Number,
+    status: {
+        type: String,
+        enum:['PAID', 'PENDING', 'CANCELLED'],
+        default:'ONGOING'

```

Comment:
Can we update default to pending as well?

Reply:
O! nice catch haha